scancodeio/settings/dev.py

```diff
@@ -32,7 +32,7 @@
 DEBUG = True
 TEMPLATES[0]["OPTIONS"]["debug"] = True
 
-ALLOWED_HOSTS = ["localhost", "127.0.0.1", "::1"]
+ALLOWED_HOSTS = ["localhost", "127.0.0.1", "::1", "0.0.0.0"]
```

Comment:
I think that "0.0.0.0" a possible security problem which opens it up to ANY IP?

Reply:
Default setup from the Docker docs https://docs.docker.com/compose/django/#define-the-project-components
This is for a development setup and not for Production.
I can add a warning in the docs.